CONTRIBUTING.adoc

```diff
@@ -86,13 +90,10 @@ export PATH=$PATH:$GOPATH/bin
         $ cd _output/local/bin/linux/amd64
         $ sudo ./openshift start
 
-+
-NOTE: Replace "linux/amd64" with the appropriate value for your platform/architecture.
-
 10.  Launch another terminal, change into the same directory you started OpenShift, and deploy the private docker registry within OpenShift with the following commands (note, the --credentials option allows secure communication between the internal OpenShift Docker registry and the OpenShift server, and the --config option provides your identity (in this case, cluster-admin) to the OpenShift server):
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
+        $ sudo chmod a+rw openshift.local.config/master/admin.kubeconfig
```

Comment:
after you run openshift with sudo and try to login or create a project from the vagrant user, this is what happens:
`
[vagrant@openshiftdev ~]$ oc login
Error loading config file "/home/vagrant/openshift.local.config/master/admin.kubeconfig": open /home/vagrant/openshift.local.config/master/admin.kubeconfig: permission denied
[vagrant@openshiftdev ~]$ ls -ltr /home/vagrant/openshift.local.config/master/admin.kubeconfig
-rw-------. 1 root root 7300 Oct 15 17:55 /home/vagrant/openshift.local.config/master/admin.kubeconfig
[vagrant@openshiftdev ~]$ sudo chmod g=r,o=r /home/vagrant/openshift.local.config/master/admin.kubeconfig
[vagrant@openshiftdev ~]$ ls -ltr /home/vagrant/openshift.local.config/master/admin.kubeconfig
-rw-r--r--. 1 root root 7300 Oct 15 17:55 /home/vagrant/openshift.local.config/master/admin.kubeconfig
[vagrant@openshiftdev ~]$ oc login
Authentication required for https://10.0.2.15:8443 (openshift)
Username: test
Password: 
Login successful.

You don't have any projects. You can try to create a new project, by running

    $ oc new-project <projectname>

error: open /home/vagrant/openshift.local.config/master/admin.kubeconfig: permission denied
[vagrant@openshiftdev ~]$ oc new-project test
error: open /home/vagrant/openshift.local.config/master/admin.kubeconfig: permission denied

i don't know exactly what happens but i will try to fix this in another pr.